### ISSUE TITLE
Add new categories to an existing model 

### ISSUE THREAD

**Issue by Rahul-Venugopal** [AlexeyAB/darknet#2357]
Hi ,

I am using yolov3.weights trained on MS COCO dataset to detect 80 classes. Is it possible to retrain the network to detect 80(old) + 2(new) classes using only training images of 2 new classes ?

**Comment by AlexeyAB** [comment]
@Rahul-Venugopal Hi,

> Is it possible to retrain the network to detect 80(old) + 2(new) classes using only training images of 2 new classes ?

You should train your model by using `yolov3.conv.81` pre-trained weights file and dataset (80(old) + 2(new)).

**Comment by Rahul-Venugopal** [comment]
Thanks for your support @AlexeyAB .

**Now I am going to use `yolov3.conv.81` pre-trained weights file** 

and it would be great if you can help me to clear my doubts 

1) Does the detection accuracy of 80 classes **remain same**  , provided I use the same old dataset for 80 (old) classes ? or is there any chance for that to change ?

2) If I add more images to some of this 80 classes , will it help to improve the detection accuracy of that particular classes ? 



**On a custom dataset** 

I have trained yolov3 on **5 classes** (custom dataset) using pretrained weights on imagenet 
( `darknet53.conv.74`) . 
My final weights are `yolov3_custom_final.weights
`
Now if I want to add 3 more classes to this model , can I use 

`./darknet partial cfg/yolov3_custom.cfg yolov3_custom_final.weights yolov3_custom.conv.5 5`

to create pretrained weights from this model ?
